WordPress/Classes/ViewRelated/Reader/ReaderPostsViewController.m

```diff
@@ -915,6 +916,8 @@ - (void)configurePostCell:(ReaderPostTableViewCell *)cell atIndexPath:(NSIndexPa
     cell.postView.shouldShowAttributionMenu = shouldShowAttributionMenu;
     cell.postView.canShowActionButtons = self.hasWPComAccount;
     cell.postView.shouldShowAttributionButton = self.hasWPComAccount;
+    
+    [self setReblogButtonVisibilityOfPostView:cell.postView];
```

Comment:
Great idea! That will also help the readability I'm sure since we can set the boolean along with all the other ones above.